packages/host/src/components/AuthenticationAlert.tsx

```diff
@@ -38,14 +39,24 @@ export const AuthenticationAlert = () => {
       ? t('auth.logged-out-message')
       : t('auth.permission-denied');
 
+  const onOk = () => {
```

Comment:
offroad- store the location when redirecting due to a lack of auth token.
not for other problems, if it was permission denied, that could be an annoying loop